impl/sql/directory/storage_test.go

```diff
@@ -23,11 +23,11 @@ import (
 	"github.com/golang/protobuf/proto"
 	"github.com/google/go-cmp/cmp"
 	"github.com/google/keytransparency/core/directory"
+	"github.com/google/trillian"
```

Comment:
nit: because the `trillian` package is a proto, we prefer to import rename it to something ending in "pb" 

```suggestion
	tpb "github.com/google/trillian"
```